pgpubsub/settings.py

```diff
@@ -0,0 +1,3 @@
+from settings import *
```

Comment:
@PaulGilmartin can you remove the settings from here and change the path back in `manage.py`? This settings will be included in your package on PyPI when deployed. I think it may even fail packaging since it is importing from the root folder

It's fine to have these settings in the base settings.py

Reply:
I updated this in a separate PR that I will open against this one

Reply:
https://github.com/Opus10/django-pgpubsub/pull/5

Reply:
@wesleykendall the reason I didn't edit the settings.py file directly was because for some reason Pycharm complained that it was not part of the project. I don't know if that was somehow an error on my side in how I pulled in the repo.